common/common.cpp

```diff
@@ -1989,6 +2021,12 @@ std::string fs_get_cache_file(const std::string & filename) {
     return cache_directory + filename;
 }
 
+void hf_get_token_from_env(gpt_params & params) {
```

Comment:
```suggestion
void hf_set_token_from_env(gpt_params & params) {
```

The function does not just get and return, but it set the value to `params.hf_token`